test/passing/sequence_preserve.ml.ref

```diff
@@ -0,0 +1,13 @@
+let foo x y =
+  do_some_setup y ;
+  important_function x
+
+let foo x y =
+  do_some_setup x ;
+  do_some_setup y ;
+  
```

Comment:
This line contains space characters. Same for the next one.

Reply:
fixed